flash/vision/classification/data.py

```diff
@@ -73,7 +75,13 @@ def __getitem__(self, index: int) -> Tuple[Any, Optional[int]]:
         filename = self.fnames[index]
         img = self.loader(filename)
         if self.transform is not None:
-            img = self.transform(img)
+            if str(type(self.transform)) == "<class 'albumentations.core.composition.Compose'>":
```

Comment:
Not a huge fan of this hard-coded check. Any better solution ?

Reply:
Agreed this was developed just so that I could get unblocked as soon as possible. Remember this pull request is not meant to be merged it's rather meant to serve as a reference as potential solutions for the open issues that need to be resolved so that Flash can be used practically for Kaggle.